x-pack/plugins/search_playground/public/components/question_input.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { css } from '@emotion/react';
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface QuestionInputProps {
+  value: string;
+  onChange: (value: string) => void;
+  button: React.ReactNode;
+  isDisabled?: boolean;
+}
+
+export const QuestionInput: React.FC<QuestionInputProps> = ({
+  value,
+  onChange,
+  button,
+  isDisabled,
+}) => {
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => onChange(e.target.value);
+
+  return (
+    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween" css={{ position: 'relative' }}>
+      <input
+        className="euiFieldText euiFieldText--fullWidth"
+        style={{
+          padding: '20px 76px 20px 20px',
```

Comment:
Doing this with hard pixel values everywhere is generally a bad idea and won't really scale. Is there really no way to do this with either EUI components? If not, we should at least use EUI size variables to derive these numbers from.

Using EUI internal classes to style components is also no longer really supported and ideally not something we're doing here. If you're using those classes and custom styling, you're probably better off just using an `EuiFieldText` component and tweaking that (ideally tweaking it very little).

Reply:
@sphilipse I found the way to override EUIFieldText styles, but it has less priority then basic styles. So is it preferable to use `!important` in that case or pass the style in styles instead of CSS field?

Reply:
I'm not sure what you mean here--we definitely shouldn't be using `important!`. You should be able to pass style in the css field of an EuiFieldText. But it's hard to give you advice because I don't quite know what the look is you're trying to achieve and what the gap is with the existing EuiFieldText component.